docs/source/install/rhel6.rst

```diff
@@ -0,0 +1,291 @@
+RHEL 6 / CentOS 6
+=================
+
+This guide provides step-by step instructions on installing StackStorm on a single box on RHEL 6/CentOS 6.
+A script `st2bootstrap-el7.sh <https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-el6.sh>`_,
+codifies the instructions below.
+
+.. warning :: Currently BETA! Please try, use and report bugs on
+   `github.com/StackStorm/st2-packages <https://github.com/StackStorm/st2-packages/issues/new>`_.
+   Soon, package-based installation will be
+   the preferred path to installing StackStorm.
+
+.. contents::
+
+
+Minimal installation
+--------------------
+
+Adjust SELinux policies
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If your RHEL/CentOS box has SELinux enforced, please follow these instructions to adjust SELinux
+policies. This is needed for successful installation. If you are not happy with these policies,
+you may want to tweak them according to your security practices.
+
+* Check if SELinux is enforcing
+
+    .. code-block:: bash
+
+        getenforce
+
+* If previous command returns enforcing, then run the following commands to adjust SELinux policies:
+
+    .. code-block:: bash
+
+        # SELINUX management tools, not available for some minimal installations
+        sudo yum install -y policycoreutils-python
+
+        # Allow rabbitmq to use '25672' port, otherwise it will fail to start
+        sudo semanage port --list | grep -q 25672 || sudo semanage port -a -t amqp_port_t -p tcp 25672
+
+        # Allow network access for nginx
+        sudo setsebool -P httpd_can_network_connect 1
+
+Install Dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Install MongoDB, RabbitMQ, and PostgreSQL.
+
+  .. code-block:: bash
+
+    sudo yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+    sudo yum -y install mongodb-server rabbitmq-server
+    sudo service mongod start
+    sudo service rabbitmq-server start
+    sudo chkconfig mongod on
+    sudo chkconfig rabbitmq-server on
```

Comment:
Sorry, all these changes can be made after merging. Let's focus on correctness of instructions first.